lightning/src/onion_message/offers.rs

```diff
@@ -44,7 +45,7 @@ pub trait OffersMessageHandler {
 	/// The returned [`OffersMessage`], if any, is enqueued to be sent by [`OnionMessenger`].
 	///
 	/// [`OnionMessenger`]: crate::onion_message::messenger::OnionMessenger
-	fn handle_message(&self, message: OffersMessage, responder: Option<Responder>) -> ResponseInstruction<OffersMessage>;
+	fn handle_message(&self, message: OffersMessage, responder: Option<Responder>, context: OffersContext) -> ResponseInstruction<OffersMessage>;
```

Comment:
Let's move the `context` parameter before the `responder` parameter. Same for the custom handler.

Reply:
Done! Fix the ordering of parameters for `handle_message` and `handle_custom_message` in [528dc64](https://github.com/lightningdevkit/rust-lightning/commit/528dc644e5b995c10f96fe36fb803bc6433580d7)
Thanks!